gg_interface/src/main.cpp

```diff
@@ -39,21 +37,27 @@ void setup() {
     ina260.MaskEnable->read();
 
     // Enable alert on "Conversion Ready"
-    ina260.MaskEnable->write(0x0400);
+    ina260.MaskEnable->write(CONVERSION_READY_ADDRESS);
 
     // Attach ISR to "alert" pin from the INA260 to handle reading of the data
-    attachInterrupt(digitalPinToInterrupt(alertPin), alert, FALLING); // TODO: FALLING or LOW?
+    attachInterrupt(digitalPinToInterrupt(ALERT_PIN), alert, FALLING); // TODO: FALLING or LOW?
+
+    initialized = true;
 }
 
 void loop() {
+    if (!initialized) {
+        return;
+    }
+
     // Das blinkenlight!
     if (ledState) {
-        digitalWrite(ledPin, LOW);
-        ledState = false;
+        digitalWrite(LED_PIN, LOW);
     } else {
-        digitalWrite(ledPin, HIGH);
-        ledState = true;
+        digitalWrite(LED_PIN, HIGH);
     }
+
+    ledState = !ledState;
     delay(100);
```

Comment:
I had already changed this around to the following, it is simpler and eliminates the state variable:
```cpp
// Das blinkenlight!
digitalWrite(ledPin, !digitalRead(ledPin));
delay(500);
```

Reply:
Cool, I'll update this!